templates/aws-standalone-cp/README.md

```diff
@@ -0,0 +1,11 @@
+## Install applications into Target Cluster
+
+To install applications into the target cluster created using Cluster API (CAPI) upon creation, a Flux `HelmRelease` object is to be made such that its `.spec.KubeConfig` references the kubeconfig of the target cluster.
+
+**Reference:** https://fluxcd.io/flux/components/helm/helmreleases/#remote-clusters--cluster-api
+
+This chart/template already defines the following applications under `templates/apps` which can be be installed into the target cluster by setting `.Values.installApps=true`:
+1. cert-manager
+2. nginx-ingress
+
+**Important:** The Flux objects added to `templates/apps` to install custom applications must have the `hmc.mirantis.com/managed: "true"` label to be reconciled by HMC.
```

Comment:
is there a specific reason why this is necessary? Maybe reference the code in HMC that needs this?

Reply:
The reason is because we instantiate Flux's controllers to watch for the `hmc.mirantis.com/managed: "true"` label:
https://github.com/Mirantis/hmc/blob/ee22e2bbf6a823b22af6a688fea86c2d0acbd873/templates/hmc/values.yaml#L71-L78

I can probably add some more context in this README.

Reply:
Ah, that makes sense, I feel the documentation is enough, I was more wondering why this is from an internal point of view :)